src/scripts/angry-tock.js

```diff
@@ -23,6 +23,9 @@ const ANGRY_TOCK_SECOND_ALERT = moment(
 const ANGRY_TOCK_REPORT_TO = (
   process.env.ANGRY_TOCK_REPORT_TO || "#18f-supes"
 ).split(",");
+const HAPPY_TOCK_REPORT_TO = (process.env.HAPPY_TOCK_REPORT_TO || "#18f").split(
+  ","
+);
```

Comment:
The line break here seems goofy to me, is this from an autoformatter?

If it were me, I'd do:

```suggestion
const HAPPY_TOCK_REPORT_TO = (process.env.HAPPY_TOCK_REPORT_TO || "#18f")
  .split(",");
```

Reply:
Yep, [prettier](https://prettier.io/) did it. 🤷‍♂️